.travis.yml

```diff
@@ -9,6 +9,7 @@ env:
 - PACKAGES="django>=1.8,<1.9"
 - PACKAGES="django>=1.10,<1.11"
 - PACKAGES="django>=1.11,<1.12"
+- PACKAGES="django==2.0"
```

Comment:
This probably wants to be a little more flexible, to make sure the tests are always run against the latest point release.

To be consistent with the other lines in this file, that would mean changing this to:

```
PACKAGES="django>=2.0,<2.1"
```

Although this syntax is more terse:

```
PACKAGES="django~=2.0.0"
```